tools/ci_build/github/linux/run_python_dockerbuild.sh

```diff
@@ -34,7 +38,7 @@ docker run --rm \
     -e ORT_DISABLE_PYTHON_PACKAGE_LOCAL_VERSION \
     -e DEFAULT_TRAINING_PACKAGE_DEVICE \
     $ADDITIONAL_DOCKER_PARAMETER \
-    $DOCKER_IMAGE tools/ci_build/github/linux/build_linux_python_package.sh -c $BUILD_CONFIG -p $PYTHON_EXES $DOCKER_SCRIPT_OPTIONS
+    $DOCKER_IMAGE tools/ci_build/github/linux/build_linux_python_package.sh $DOCKER_SCRIPT_OPTIONS
```

Comment:
Do we need to remove `-c $BUILD_CONFIG`?

Reply:
Yes, it is already being added above in line 19. This was adding it twice. Here's a snippet of the command args in the failing pipeline:
![image](https://github.com/user-attachments/assets/e20599d8-0b63-4df7-8bb6-58d111d52c90)